pages/analyze_impacts.py

```diff
@@ -802,7 +811,7 @@ def validate_date(feature):
             dam_batch_fc = ee.FeatureCollection(dam_batch)
 
             # Process batch through pipeline
-            s2_cloud_mask_batch = ee.ImageCollection(s2_export_for_visual(dam_batch_fc))
+            s2_cloud_mask_batch = ee.ImageCollection(s2_export_for_visual_generic(dam_batch_fc, add_elevation_band))
```

Comment:
this is a huge improvement, but i would have kept the name `s2_export_for_visual` since the 'generic' is now a bit redundant